src/search.ts

```diff
@@ -110,7 +110,13 @@ export default class Search extends SearchUtils implements SearchInterface {
             return;
         }
         libSymphonySearch.symSEDestroy();
-        libSymphonySearch.symSEInit();
+        try {
+            libSymphonySearch.symSEInit(searchConfig.LIBRARY_CONSTANTS.DICTIONARY_PATH);
+        } catch (e) {
+            logger.error(`search: Initialization failed (symSEInit)`, e);
+            this.setLibInitState(false);
```

Comment:
does swift search fail entirely if we have an exception here?

Reply:
@VishwasShashidhar Yes, SwiftSearch will be disabled.

I will be testing this in the custom build. Will not be able to test this in dev env.